src/data/queries/index.ts

```diff
@@ -37,6 +38,7 @@ export const QUERIES_MAP = {
   'paragraph--link_teaser': LinkTeaser,
   'paragraph--rich_text_char_limit_1000': Wysiwyg,
   'paragraph--wysiwyg': Wysiwyg,
+  'paragragh--alert_single': AlertSingle,
   'media--image': MediaImage,
   'block--alert': Alert,
```

Comment:
this `block--alert` feels very similar to the alertSingle query you've got, can it be refactored to use the same `transformAlert` function you've added in `alertSingle`? or removed if it's no longer needed?

Reply:
I refactored them both to use the transformAlert function. It feels natural to leave them separate in case there are further modifications needed in the future